lib/ProMotion/ipad/split_screen.rb

```diff
@@ -16,15 +16,35 @@ def create_split_screen(master, detail, args={})
 
     # UISplitViewControllerDelegate methods
 
-    def splitViewController(svc, willHideViewController: vc, withBarButtonItem: button, forPopoverController: pc)
+    # iOS 7 and below
+    def splitViewController(svc, willHideViewController: vc, withBarButtonItem: button, forPopoverController: _)
+      button ||= self.displayModeButtonItem if self.respond_to?(:displayModeButtonItem)
+      return unless button
       button.title = @pm_split_screen_button_title || vc.title
       svc.detail_screen.navigationItem.leftBarButtonItem = button
     end
 
-    def splitViewController(svc, willShowViewController: vc, invalidatingBarButtonItem: barButtonItem)
+    def splitViewController(svc, willShowViewController: _, invalidatingBarButtonItem: _)
       svc.detail_screen.navigationItem.leftBarButtonItem = nil
     end
```

Comment:
Minor updates to these two methods, since we're forwarding to them in iOS 8 (they're no longer called automatically).